README.md

```diff
@@ -1,53 +1,57 @@
-| `CUSDEB_API_URL` | The URL of CusDeb API, including its version. The value of the param *must be* `/api/v1` when running locally. |
-| `HOST`           | The address of the machine Webpack Dev Server is running on. It helps to check CusDeb Web Client on other devices in a local network. |
+| Parameter              | Description                                                                                                                                         | Default                                     |
+|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------|
+| `BLACKMAGIC_URL`       | The URL of the WebSocket API provided by [Black Magic](https://github.com/tolstoyevsky/blackmagic).                                                 | `ws://localhost:8002/bm/token/%token`       |
+| `CUSDEB_ANONYMOUS_URL` | The URL of [CusDeb Anonymous](https://github.com/tolstoyevsky/cusdeb-anonymous).                                                                    | `http://localhost:8007`                     |
+| `CUSDEB_API_URL`       | The URL of [CusDeb API](https://github.com/tolstoyevsky/cusdeb-api).                                                                                | `http://localhost:8001`                     |
+| `CUSDEB_HELPIK_URL`    | The URL of [CusDeb Helpik](https://github.com/tolstoyevsky/cusdeb-helpik).                                                                          | `http://localhost:8005`                     |
+| `CUSDEB_TZ_URL`        | The URL of [CusDeb Tz](https://github.com/tolstoyevsky/cusdeb-tz).                                                                                  | `http://localhost:8006`                     |
+| `DOMINION_URL`         | The URL of the WebSocket API provided by [Dominion](https://github.com/tolstoyevsky/dominion).                                                      | `ws://localhost:8003/dominion/token/%token` |
+| `HOST`                 | The address of the machine Webpack Dev Server is running on. It helps to check CusDeb Web Client on other devices in a local network.               | `0.0.0.0`                                   |
+| `MODE`                 | The webpack configuration option tells webpack to use its built-in optimizations accordingly when it's sets to `development`.                       | `development`                               |
+| `PORT`                 | The port which Webpack Dev Server is running on.                                                                                                    | `8000`                                      |
+| `SENTRY_DSN`           | A [Sentry](http://sentry.io/) DSN (Data Source Name) tells a Sentry SDK where to send events so the events are associated with the correct project. |                                             |
```

Comment:
Let's paraphrase the description: `A [Sentry](http://sentry.io/) DSN (Data Source Name) to tell the Sentry SDK where to send events.`